consensus/src/checkpoint.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::common::{Chain, ProtocolStateChain};
+use mina_crypto::base58::Base58Encodable;
+use mina_crypto::hash::{EpochSeed, StateHash};
+use mina_rs_base::protocol_state::ProtocolState;
+#[allow(dead_code)]
```

Comment:
I don't think we need this. This is being used below right?

Reply:
We use this in the init_checkpoints()

Reply:
then this attribute should not be there, because the lint says dead_code

Reply:
If we don't use this tag these Clippy warnings will come out again

Reply:
It actually used at [here ](https://github.com/ChainSafe/mina-rs/blob/a69a2d09029df9ef739a4d4a9e77d8b7b014b39b/consensus/src/checkpoint.rs#L62)but Clippy think it's never used [Github Action](https://github.com/ChainSafe/mina-rs/runs/4284862218?check_suite_focus=true#step:9:20)

Reply:
ok, let's tackle this later.